pkg/cmd/admin/registry/registry.go

```diff
@@ -181,6 +186,8 @@ func NewCmdRegistry(f *clientcmd.Factory, parentName, name string, out, errout i
 	cmd.Flags().StringVar(&cfg.Selector, "selector", cfg.Selector, "Selector used to filter nodes on deployment. Used to run registries on a specific set of nodes.")
 	cmd.Flags().StringVar(&cfg.ServingCertPath, "tls-certificate", cfg.ServingCertPath, "An optional path to a PEM encoded certificate (which may contain the private key) for serving over TLS")
 	cmd.Flags().StringVar(&cfg.ServingKeyPath, "tls-key", cfg.ServingKeyPath, "An optional path to a PEM encoded private key for serving over TLS")
+	cmd.Flags().StringSliceVar(&cfg.SupplementalGroupRange, "supplemental-groups", cfg.SupplementalGroupRange, "Specify supplemental groups which is an array of ID's that grants group access to registry shared storage")
```

Comment:
`Range` in the name is a bit confusing. As far I can see, it's just a list and not range like 10-15 or 10/5.

The code for parsing `fsGroup`/`supplementalGroups` from the annotations that actually support ranges could be found here:
- https://github.com/openshift/origin/blob/bd3e362de2a9f849bd2d7d3ff2a372746f35fb2a/pkg/security/scc/matcher.go#L334-L393
- https://github.com/openshift/origin/blob/bd3e362de2a9f849bd2d7d3ff2a372746f35fb2a/pkg/security/uid/uid.go#L18-L40